### DISCUSSION TITLE
Win 3 CA project to Win 4.0.4 fails installing nuget

### DISCUSSION THREAD

**Discussion by bkblie** [wixtoolset/issues#8045] [Questions] [answered]
Cant add the nuget https://www.nuget.org/packages/wix/4.0.4/ to my project.
Is there anything i can do to fix this?

**Output from Package Manager:**
Attempting to gather dependency information for package 'wix.4.0.4' with respect to project 'Installers\MyCustom.CA', targeting '.NETFramework,Version=v4.8.1'
Gathering dependency information took 296 ms
Attempting to resolve dependencies for package 'wix.4.0.4' with DependencyBehavior 'Lowest'
Resolving dependency information took 0 ms
Resolving actions to install package 'wix.4.0.4'
Resolved actions to install package 'wix.4.0.4'
Install failed. Rolling back...
Package 'wix.4.0.4' does not exist in project 'MyCustom.CA'
Package 'wix.4.0.4' does not exist in folder 'C:\code\testing\packages'
Executing nuget actions took 61 ms
Package 'wix 4.0.4' has a package type 'DotnetTool' that is not supported by project 'Installers\MyCustom.CA'.
Time Elapsed: 00:00:00.3659951

**Comment by robmen** [comment]
Please read the Getting Started documentation at https://wixtoolset.org. Acquiring WiX v4 is very different from WiX v3.

**Reply by bkblie** [reply, marked as answer]
I made a new Custom Action project to my solution from the Heatwave template.
This worked, and copyed all my [CustomAction] code into the new procect.

Wile installing via my .msi file the installer cant fint my CA.dll

Is there also a another way to include the CA.dll into my MSI installer in WIX 4.0.4
